test/e2e_node/runtime_conformance_test.go

```diff
@@ -45,8 +45,12 @@ var _ = framework.KubeDescribe("Container Runtime Conformance Test", func() {
 			It("it should run with the expected status [Conformance]", func() {
 				restartCountVolumeName := "restart-count"
 				restartCountVolumePath := "/restart-count"
+				priv := true
 				testContainer := api.Container{
 					Image: ImageRegistry[busyBoxImage],
+					SecurityContext: &api.SecurityContext{
+						Privileged: &priv,
+					},
```

Comment:
In this test, data is being written to a file in /tmp, so privileged.

However, i think that another way would be to use emptydir with selinux labels, and it should work across container restart (as this test checks for restarts) I think.